docs/src/integrations/source/troubleshooting.md

```diff
@@ -0,0 +1,40 @@
+---
+title: "Troubleshooting"
+description: In this page you'll learn how to troubleshoot problems related to access rights when cloning repositories.
+---
+
+## Unable to clone a repository
+
+If you [add a user](/administration/users.md#add-a-user-to-a-project) with the **Project Admin** or **Contributor** role to a project on Platform.sh,
```

Comment:
```suggestion
If you [add a user](/administration/users.md#add-a-user-to-a-project) to a project on Platform.sh,
```

The text currently implies it's only those two roles, but it's actually also Admins for an Environment type and I don't think it's worth spending enough space here to explain all of that. (That's the problem with have a role Project Admin and another as Admin -- they're easily confused.) So I'd suggest not getting into it.